### PR TITLE
[SCH-130] Removes left over symbolic link and updates travis yaml to latest standards

### PR DESCRIPTION
**Pull request by lobsterdore**
Also removed references to ansible-city

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,13 +15,18 @@ branches:
     - develop
     - master
 
+env:
+  - ANSIBLE_INSTALL_VERSION=1.9.6
```
Comment (racer1680) [inline .travis.yml]: 1.9.4 is the version I think :)

File: .travis.yml [hunk 2]
```diff
@@ -8,13 +15,18 @@ branches:
...
+  - ANSIBLE_INSTALL_VERSION=2.0.0.2
```
Comment (racer1680) [inline .travis.yml]: Ansible 2.0? Are you sure? Who will be able to support it? Who will be able to reuse it?

File: .travis.yml (hunk 1 above)
Comment (wojtek-oledzki) [inline .travis.yml]: 1.9.6 is the latest 1.9 version (http://releases.ansible.com/ansible/)

File: .travis.yml (hunk 2 above)
Comment (wojtek-oledzki) [inline .travis.yml]: I'm using the latest 2.0 in other projects ;)

**Comment by wojtek-oledzki** [comment]
Looks good to me :tm:

LGTM

**Pull request merged**